applicator.json

```diff
@@ -0,0 +1,60 @@
+{
+    "$schema": "http://json-schema.org/draft-08/schema#",
+    "$id": "https://json-schema.org/draft-08/applicator",
+    "$recursiveAnchor": true,
+    "$vocabulary": {
+        "https://json-schema.org/draft-08/vocabularies/applicator": true
+    },
+
+    "title": "Applicator vocabulary meta-schema",
+    "$defs": {
+        "schemaArray": {
+            "type": "array",
+            "minItems": 1,
```

Comment:
Should `minItems` be defined in this vocabulary or it should be included with another vocabulary?

Reply:
@vearutop it is included in another vocabulary, specifically the assertions vocabulary defined in the Validation spec.  Note that this document, _applicator_, still uses _schema_ as its meta-schema.  Once I add `$vocabulary` to _schema_ it will reference the assertion vocabulary, making it available for use in _applicator_.

This is why _applicator_ is not its own meta-schema.  It uses more keywords than it defines.

Reply:
Marked resolved because I assume that with no further reply for two weeks that my answer was sufficient :-)